mobile/ios/device/ios-core.ts

```diff
@@ -749,23 +751,27 @@ export class WinSocket implements Mobile.IiOSDeviceSocket {
 
 	public close(): void {
 		this.winSocketLibrary.closesocket(this.service);
+		if (this.sysLogChildProcess) {
+			this.sysLogChildProcess.kill();
```

Comment:
You can consider assigning `this.sysLogChildProcess = null;` after killing the process